### PR TITLE
Add support for more currencies

### PR DESCRIPTION
**Pull request by zaidhaan**
Hey there, nice tool! I've noticed that not all the currencies from the [XML file](https://www.ecb.europa.eu/stats/eurofxref/eurofxref-daily.xml) you guys get your exchange rate data from are used (some of which I use quite frequently), so I thought I'd add a few of the remaining ones.

Not entirely sure if this is correct, but I did some testing and it looks alright.

### CODE REVIEW

File: numbat/modules/units/currencies.nbt [hunk 1]
```diff
@@ -28,3 +28,51 @@ unit canadian_dollar: Money = EUR / exchange_rate("CAD")
...
+unit south_korean_won: Money = EUR / exchange_rate("KRW")
+
+@aliases(malaysian_ringgits, MYR: short, RM)
+unit malaysian_ringgit: Money = EUR / exchange_rate("MYR")
+
+@aliases(new_zealand_dollars, NZD: short, NZ$)
+unit new_zealand_dollar: Money = EUR / exchange_rate("NZD")
+
+@aliases(philippine_pesos, PHP: short, ₱)
+unit philippine_peso: Money = EUR / exchange_rate("PHP")
```
Comment (mhmdanas) [inline numbat/modules/units/currencies.nbt:72]: Let's add the Saudi Riyal as well:

```
@aliases(saudi_riyals, SAR: short, SR)
unit saudi_riyal: Money = EUR / exchange_rate("SAR")
```

File: numbat/src/lib.rs [hunk 2]
```diff
@@ -332,6 +332,69 @@ impl Context {
...
+                        "MYR",
+                        "RM",
+                        "new_zealand_dollar",
+                        "new_zealand_dollars",
+                        "NZD",
+                        "NZ$",
+                        "philippine_peso",
+                        "philippine_pesos",
+                        "PHP",
+                        "₱",
```
Comment (mhmdanas) [inline numbat/src/lib.rs:389]:
```suggestion
                        "₱",
                        "saudi_riyal",
                        "saudi_riyals",
                        "SAR",
                        "SR",
```

**Review by mhmdanas — CHANGES_REQUESTED** [review]
Thank you! Can you also please add the new currencies to `book/src/list-units.md`?

**Comment by sharkdp** [comment]
> Can you also please add the new currencies to `book/src/list-units.md`?

Yes, that would be great. I want to automate this at some point, but didn't get around to it.

**Comment by sharkdp** [comment]
> I want to automate this at some point, but didn't get around to it.

Did that now in #223 and updated your PR with the corresponding information.

**Review by sharkdp — APPROVED** [review]

**Comment by sharkdp** [comment]
Changes are live on numbat.dev: https://numbat.dev/?q=%E2%82%AC+-%3E+%E0%B8%BF%E2%8F%8E

**Comment by zaidhaan** [comment]
Ah, thanks for completing that @sharkdp! Sorry, was a little bit busy lately and wasn't able to get to it myself

**Pull request merged**